.github/workflows/php_tests.yml

```
@@ -62,13 +62,17 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           php-version: '8.2'
+      - name: Get WC pinned pnpm version
+        id: get_wc_pnpm_version
+        run: echo "WC_PNPM=$(node -p \"require('/tmp/woocommerce/plugins/woocommerce/package.json').engines.pnpm\")" >> "$GITHUB_OUTPUT"
```

Sorry, my bad! I've just learned that using `$()` creates a new quoting context. So this should work (it does when I test it locally after defining `GITHUB_OUTPUT`):

```suggestion
        run: echo "WC_PNPM=$(node -p "require('/tmp/woocommerce/plugins/woocommerce/package.json').engines.pnpm")" >> "$GITHUB_OUTPUT"
```